src/Handler/CurlMultiHandler.php

```diff
@@ -20,11 +20,30 @@ class CurlMultiHandler
 {
     /** @var CurlFactoryInterface */
     private $factory;
+
     /** @var int */
     private $selectTimeout;
-    private $active;
+
+    /** @var resource|null the currently executing resource in `curl_multi_exec`. */
+    private $active = null;
+
+    /**
+     * @var array Request entry handles, indexed by handle id in `addRequest`.
```

Comment:
Maybe this should be an array of resources ? no?

Reply:
When looking at `addRequest` it's values aren't resources. `$handles` is an array of arrays.

```
# line 121
$this->addRequest(['easy' => $easy, 'deferred' => $promise]);
```

Reply:
true.. sorry..